src/main/java/com/ajou/hertz/domain/user/controller/UserController.java

```diff
@@ -117,6 +122,24 @@ public ResponseEntity<UserResponse> signUpV1(
 			.body(UserResponse.from(userCreated));
 	}
 
+	@Operation(
+		summary = "프로필 이미지 변경",
+		description = "프로필 이미지를 변경합니다.",
+		security = @SecurityRequirement(name = "access-token")
+	)
+	@PutMapping(
+		value = "/me/profile-images",
+		headers = API_VERSION_HEADER_NAME + "=" + 1,
+		consumes = MediaType.MULTIPART_FORM_DATA_VALUE
+	)
+	public UserResponse updateProfileImageV1(
+		@AuthenticationPrincipal UserPrincipal userPrincipal,
+		@RequestPart("profileImage") MultipartFile profileImage
```

Comment:
```suggestion
		@RequestPart MultipartFile profileImage
```

Request parameter로 받고자 하는 이름은 기본적으로 변수명과 같게 설정되기 때문에, 특별히 명시하시려고 하신 게 아니라면 뺴도 괜찮습니다.